pandas/core/internals/construction.py

```diff
@@ -536,9 +536,17 @@ def _list_of_series_to_arrays(data, columns, coerce_float=False, dtype=None):
 
 
 def _list_of_dict_to_arrays(data, columns, coerce_float=False, dtype=None):
+    """Convert list of OrderedDict to dict
```

Comment:
```suggestion
    """Convert list of dicts to arrayst
```

the output are arrays? (or did I misinterpret your sentence?)

Reply:
You're right, I'm just getting impatient.